build.assets/tooling/cmd/release-notes/template/release-notes.md.tmpl

```diff
@@ -0,0 +1,20 @@
+## Description
```

Comment:
Somebody from docs team should review this. The downloads page pulls from the GH release notes and parses it, so any changes to the structure/formatting should be reviewed by them.

Reply:
It's fine, it's the same format as we've always used. We can tweak later if needed.